translator/translate/logs/logs_collected/windows_events/collect_list/collectlist_test.go

```diff
@@ -59,11 +59,11 @@ func TestApplyRule(t *testing.T) {
 
 	var actual interface{}
 
-	error := json.Unmarshal([]byte(rawJsonString), &input)
-	if error == nil {
+	e := json.Unmarshal([]byte(rawJsonString), &input)
```

Comment:
This is not consistent with all of the other name changing. Why is this not `err`?

Reply:
Fixed it in the newest commit. Nice catch on these though.